playbooks/azure/openshift-cluster/build_base_image.yml

```diff
@@ -67,3 +68,15 @@
       openshift_azure_vm_host: "{{ groups['azure'][0] }}"
       openshift_azure_os_tags: "{{ hostvars[groups['azure'][0]]['openshift_azure_os_tags'] | default({}) }}"
       openshift_azure_image_name: "{{ openshift_azure_base_image }}"
+      openshift_azure_vm_name: "{{ openshift_azure_base_image }}"
+
+- name: "Remove instance used to build image"
+  hosts: localhost
+  tasks:
+  - name: "rm vm"
+    import_role:
+      name: openshift_azure
+      tasks_from: remove_instance.yml
+    vars:
+      openshift_azure_vm_name: "{{ openshift_azure_base_image }}"
+      openshift_azure_vm_keep_storage: false
```

Comment:
I don't think we have any storage on the base vm.  Not sure if this matters.

Reply:
This includes base and data disk, this is what's left after the image is built:
![screenshot-2018-4-9 microsoft azure](https://user-images.githubusercontent.com/3899107/38501017-40120e90-3c0c-11e8-89a5-d7e8ce282210.png)